README.md

```diff
@@ -1,258 +1,2 @@
-* **VIRTUAL_HOST** - This is used by the nginx server as the virtual host under which the services are made avialble. This should be a host, for example: example.openlmis.org
-* **PUBLIC_URL** - The public url of the OpenLMIS distribution. Will be used in generated links pointing to this distribution. If this variable is not set, the BASE_URL will be used for the generated links. We extract this usage of BASE_URL to another environmental variable because in more complex deployments the BASE_URL does not have to be the base domain name. This should be an url, for example: https://example.openlmis.org
-* **CONSUL_HOST** - Identifies the IP address or DNS name of the Consul server. Set this to the host or IP under which the distribution is available and Consul listens for connections. Services should register with Consul under this address. This should be a host or an IP, for example 8.8.8.8.
-* **CONSUL_PORT** - The port used by the Consul server - services should use this port to register with Consul. This should be a port number, for example 8500. 8500 is used by default.
-* **REQUIRE_SSL** - Whether HTTPS is required. If set to `true`, nginx will redirect all incoming HTTP connections to HTTPS. By default SSL will not be required - either leave it blank or set to `false` if you wish to allow HTTP connections.
-* **LOCALE** - Default localized system language. It will be applied to all running services, if this variable is missing default "en" value will be used.
-* **CORS_ALLOWED_ORIGINS** - Comma-separated list of origins that are allowed, for example: `https://test.openlmis.org,http://some.external.domain`. `*` allows all origins. Leave empty to disable CORS.
-* **CORS_ALLOWED_METHODS** - Comma-separated list of HTTP methods that are allowed for the above origins.
-
-These variables are used by services for their connection to the database (none of these have defaults):
-
-* **DATABASE_URL** - The JDBC url under which the database is accessible. Our services use `jdbc:postgresql://db:5432/open_lmis` for connecting to the PostgreSQL database running in a container.
-* **POSTGRES_USER** - The username of the database user that the services should use. This variable is also used by our PostgreSQL container to create a user.
-* **POSTGRES_PASSWORD** - The password of the database user that the services should use. This variable is also used by our PostgreSQL container to create a user.
-
-These variables are used by our builds in order to integrate with the [Transifex](https://www.transifex.com/) translation management system:
-
-* **TRANSIFEX_USER** - The username to use with Transifex for updating translations.
-* **TRANSIFEX_PASSWORD** - The password to use with Transifex for updating translations.
+# OpenLMIS Selv v3 DHIS2 Integration Service
+Service for integrating SELV3 OpenLMIS implementation with DHIS2.
```

Comment:
It would be worth to add a note how the docker image can be created to use it locally with disto

Reply:
Added note about docker image creation in upcoming commit